### DISCUSSION TITLE
Spring Sleuth  No Data

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11716] [General]
Spring Sleuth  No Data,Why?skywalking version 9.7.0

<img width="1581" alt="企业微信截图_f2b8bf49-b416-43d5-89c4-023371df035b" src="https://github.com/apache/skywalking/assets/31468018/91bb36c0-0b6f-49de-9b03-a976bbbca582">

**Comment by wu-sheng** [comment]
That is not out of box, because it is duplicated with many agent plugins.

There are two options,

https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/application-toolkit-micrometer/

https://skywalking.apache.org/docs/skywalking-java/next/en/setup/service-agent/java-agent/application-toolkit-micrometer-1.10/

**Reply by 1450793561** [reply]
Some indicator data is empty，how i resolve？

<img width="1858" alt="企业微信截图_713c52c3-8ca8-4244-a40b-a7295830dc7c" src="https://github.com/apache/skywalking/assets/31468018/99f8c6b0-e7a1-4802-b91d-fe960f994ae4">

**Reply by wu-sheng** [reply]
They are from https://github.com/apache/skywalking/blob/c3113b65f67dfad4a25e06bca4d24e6969ed82bd/oap-server/server-starter/src/main/resources/meter-analyzer-config/spring-micrometer.yaml#L19-L28

I don't know whether your sleuth reports them or not.

**Reply by 1450793561** [reply]
In skywalking 9.7.0, these configurations are available by default after deployment，Is there anything else you need to configure?

<img width="1014" alt="企业微信截图_c90ac220-d5fb-46e6-ba04-b2fbae2c9018" src="https://github.com/apache/skywalking/assets/31468018/f96610e3-5087-4dcd-9762-d0bf4e7eb0a0">

**Reply by wu-sheng** [reply]
All OAP side is open by default. So, I asked whether agent side is reported? I don't know how you integrate with Sleuth, and what metrics actually get reported.
